Library/Formula/zeromq.rb

```diff
@@ -19,6 +17,18 @@ class Zeromq < Formula
     depends_on "libtool" => :build
   end
 
+  stable do
+    url "http://download.zeromq.org/zeromq-4.0.5.tar.gz"
+    sha1 "a664ec63661a848ef46114029156a0a6006feecd"
+
+    patch do
+      # enable --without-libsodium on libzmq < 4.1
+      # zeromq/zeromq4-x#105
+      url "https://gist.githubusercontent.com/minrk/478aab66adf7016158ff/raw/b5ea2d61c3f66db6ff3e266b76d1bec4ad4a238b/without-libsodium.patch"
+      sha1 "68543ff1b0f64b22994cb13b4d24bce8f76cf431"
```

Comment:
Sorry to play follow-the-bouncing-patch but can you point this to https://github.com/zeromq/zeromq4-x/commit/3d353659f716b2545f124444511d6aa4f277f336.diff please?